au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml

```diff
@@ -62,8 +62,13 @@ StyledPopupView {
         }
     }
 
-    function show() {
-        openTimer.restart()
+    function show(noDelay = false) {
+        if(noDelay) {
+            open();
+        }
+        else {
+            openTimer.restart()
+        }
```

Comment:
Just a minor nitpick about formatting to match overall QML style we have:
```qml
        if (noDelay) {
            open();
        } else {
            openTimer.restart()
        }
```

Reply:
also space after `if` and without `;` :)

Reply:
ohh, someone has to write a style checker for qml, i cant stop putting semicolons at the end of the line